apps/server/.env.example

```diff
@@ -22,7 +22,7 @@ MxApiSecret=""
 
 # Port: The port where the server runs
 PORT=8080
-HostUrl="http://localhost:8080"
+HOSTURL="http://localhost:8080"
```

Comment:
Fix case. We should fix all ENV vars to align with the convention that ENV vars should be upper-case. This also aligns with vars added to GH, which forces upper-case.